scalableminds/webknossos
# Fix getNullBucket crash and remove dead code

### URL of deployed dev instance (used for testing):
- https://hotfixgetNullBucketcrash.webknossos.xyz

### Steps to test:
- create a new annotation and restrict the mags during creation (e.g., to one mag)
- open the annotation --> it should not crash

### Issues:
- follow-up for #6542
- see https://scm.airbrake.io/projects/95438/groups/3398209786679717674/notices/3398729977235811716?tab=notice-detail

------
(Please delete unneeded items, merge only when none are left open)
